hasura.planx.uk/metadata/tables.yaml

```diff
@@ -220,6 +220,46 @@
         template_engine: Kriti
         url: '{{$base_url}}/webhooks/hasura/send-slack-notification'
         version: 2
+- table:
+    name: feedback
+    schema: public
+  object_relationships:
+    - name: flow
+      using:
+        foreign_key_constraint_on: flow_id
+    - name: team
+      using:
+        foreign_key_constraint_on: team_id
+  insert_permissions:
+    - role: public
```

Comment:
Also - I think making this just public-writable for now is the right approach. 

We can scope read permissions when to come to that functionality 👍

Reply:
Awesome! Sometimes the more precise stuff feels easier when applied via trial and error 😅

Reply:
We maybe have discussed but I don't recall `hasura.planx.uk/tests` although that say more about my memory rather than anything else. I'll have a look and if I'm bamboozled I'll take you up on that kind offer of a demo/handover 😊

Reply:
I've had a go at adding tests for the new tables. I think they'll need to change when permissions are figured out later on but I think they match the state of the ables at the moment.